packages/admin/admin-rte/src/core/translation/TranslationDialog.tsx

```diff
@@ -0,0 +1,54 @@
+import { messages } from "@comet/admin";
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, Grid } from "@mui/material";
+import { EditorState } from "draft-js";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import Rte from "../Rte";
+import RteReadOnly from "../RteReadOnly";
+
+interface TranslationDialogProps {
+    open: boolean;
+    close: () => void;
+    originalText: EditorState;
+    translatedText: EditorState;
+    editTranslation: (newValue: EditorState) => void;
+    applyTranslation: (newValue: EditorState) => void;
```

Comment:
Callbacks should start with on: `onClose`, `onEditTranslation`, `onApplyTranslation`

Reply:
bcfc964d89a1e86bd5a69898d75a41ecd600d8c8